wrappers/parking_lot/parking_lot_impl/src/lib.rs

```diff
@@ -0,0 +1,149 @@
+//! This is the "impl" crate implementing [parking_lot] support for [`Shuttle`].
```

Comment:
"This is the Shuttle replacement for the [parking_lot] crate, for use when testing with the Shuttle concurrency checker"